Dokan/Templates/colors.stencil

```diff
@@ -0,0 +1,12 @@
+import UIKit.UIColor
+
+// this is automatic generated file please don't edit it 🔫
+// MARK: - Colors
+
+extension UIColor {
+    {% for color in catalogs.first.assets %}
+      static var {{color.name|swiftIdentifier:"pretty"|lowerFirstWord|escapeReservedKeywords}}: UIColor {
```

Comment:
```suggestion

      static var {{color.name|swiftIdentifier:"pretty"|lowerFirstWord|escapeReservedKeywords}}: UIColor {
```

This will allow the result to look like 
```swift
extension UIColor {

      static var black: UIColor {
        UIColor(named: "Black")!
      } 

      static var white: UIColor {
        UIColor(named: "White")!
      } 
  }
```